automod/rules/spammentions.go

```diff
@@ -0,0 +1,38 @@
+package rules
+
+import (
+	appbsky "github.com/bluesky-social/indigo/api/bsky"
+	"github.com/bluesky-social/indigo/automod"
+)
+
+var _ automod.PostRuleFunc = SpamMentionsRule
+
+var mentionHourlyThreshold = 20
```

Comment:
FWIW, my intuition is that we'll need to bump this a lot. one example is follow-friday type threads where people list out a bunch of recommended follows, easily a few dozen.

in theory we could do some data science, but I think what probably works best is what you're doing here implicitly: start with just a "flag" which will result in a slack notification, and we can see how legit the hits are.

one workflow thing that will need improvement is flag naming. right now once a flag gets set, it sticks around forever. if we change the theshold, or re-write the rule entirely, it would make sense to purge all the old flags. we could use manually process to work around this (prefix new flags with "dev-" or "beta-"), but there are probably better options.

Reply:
Hm, daily numbers might be useful too then.  Follow-friday behaviors could tip the hourly count is that number is relatively low, but if we also do a daily number that's considerably higher, and it should be easy for follow-friday to stay below that while real spam still goes above it.

A mechanism for counting hours-today-that-crossed-the-line would also buff that out.  Follow-friday shouldn't go above 1 or 2.